### ISSUE TITLE
Configuration option to enforce the use of the raw value of the Authorization header.

### ISSUE THREAD

**Issue by AdrianVasiliu** [getindata/flink-http-connector#76]
As [documented](https://github.com/getindata/flink-http-connector/blob/main/README.md#basic-authentication), the connector currently processes the value set for the `Authorization` header as follows: it encodes it to Base64 and it prefixes it with "Basic ". The only case when this is not done is if the value is already prefixed with "Basic ".
This is convenient for Basic Authentication, as it allows writing
```
'gid.connector.http.source.lookup.header.Authorization' = 'username:password'
```
without taking care of b64 and prefixing.

However, there is currently no way to enforce the use of the raw value of the `Authorization` header. This is necessary for use-cases where the called API needs the header to be set for other purposes than Basic Authentication, such as API key authentication.

**Comment by kristoffSC** [comment]
Hi @AdrianVasiliu I have merged your PR,
There is one more PR that I would like to take a look before releasing the new version.

There is also a  PR where we will drop 1.15 Flink support but this will have its own version.

**Comment by AdrianVasiliu** [comment]
Thanks a lot @kristoffSC . The other PR is from my colleague, we need both, so releasing both together would be just fine :-)

**Comment by kristoffSC** [comment]
Released in 0.12.0.
Version is already pushed to maven central. It usually takes some time until it will be visible on connector's maven page but it should be available to download from maven already

**Comment by AdrianVasiliu** [comment]
@kristoffSC Thanks! It's available now.
